ch_integration_tests/run_tests.sh

```diff
@@ -13,16 +13,16 @@ if [ ! -f "$FW" ]; then
     popd
 fi
 
-FOCAL_OS_IMAGE_NAME="focal-server-cloudimg-amd64-custom-20210106-1.qcow2"
-FOCAL_OS_IMAGE_URL="https://cloudhypervisorstorage.blob.core.windows.net/images/$FOCAL_OS_IMAGE_NAME"
+FOCAL_OS_IMAGE_NAME="focal-server-cloudimg-amd64.img"
+FOCAL_OS_IMAGE_URL="https://cloud-images.ubuntu.com/focal/current/$FOCAL_OS_IMAGE_NAME"
```

Comment:
This is much slower to download from than our bucket.

Reply:
It's only 12s slower. I took https://cloud-hypervisor-jenkins.westus.cloudapp.azure.com/blue/organizations/jenkins/libvirt/detail/PR-77/1/pipeline as a reference and it took 18s to download the image versus 30s for this PR.
The time difference is due to the image being slower to boot.

Reply:
Uh, that surprises me. Let's keep an eye on it to make sure we don't get rate limited.